server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss

```diff
@@ -109,6 +113,33 @@ See the radio-option below as an example.
   align-items: center;
 }
 
+.cf-prose-h2 {
+  @include u-font('sans', 'xl');
+  @include u-line-height('sans', 2);
+  font-weight: font-weight('bold');
+}
+
+.cf-prose-intro {
+  @include u-font('body', 'lg');
+  @include u-line-height('body', 5);
+  font-weight: font-weight('normal');
+  min-height: 36px; // Not USDWS standard
```

Comment:
Just to be 100% sure, for anything that isn't USDWS did we double check? (Just to make sure it's not a mistake in the mocks). This is kind of a meta-comment, as it'd be great if we could avoid writing as much custom theming as possible.

Reply:
Updated based on Sijia's suggestion to use `usa-prose`